kuma/javascript/src/search-results-page.jsx

```diff
@@ -96,6 +103,12 @@ export default function SearchResultsPage({ locale, query, data }: Props) {
                                     <div css={styles.summary}>
                                         {hit.summary}
                                     </div>
+                                    <div
+                                        css={styles.excerpt}
+                                        dangerouslySetInnerHTML={{
+                                            __html: hit.excerpt
```

Comment:
Is `hit.except` always a truthy thing? Just wondering what would happen if it's `null`. Then you get an empty useless div.